ShanoirChallengeMigrator/.project

```diff
@@ -20,4 +20,15 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>

```

Comment:
Hi @julien-louis, same here: I am not sure, if this file should be part of the PR? It might be good to remove it?